Screens/NoAccount.js

```diff
@@ -29,16 +28,31 @@ class NoAccount extends React.Component {
         <Paragraph style={[styles.centerText, { color: colors.primary }]}>
           Sign In With A Different Email
         </Paragraph>
-        <RoundButton
-          onPress={() => this.props.navigation.navigate("CreateProfile", {})}
-          text="Visit Healthyworks Support"
-        />
+        <Button
+          raised
+          dark
+          primary
+          style={styles.roundButtonView}
+          onPress={() => this.props.navigation.navigate('CreateProfile')}
+        >
+          <Text uppercase={false} style={styles.buttonText}>Visit Healthyworks Support</Text>          
```

Comment:
It looks like you're aware, but just pointing out that if you pass in a component to `Button` that you'll have to do all of the styling and stuff yourself.